examples/new/simple/main.go

```diff
@@ -0,0 +1,67 @@
+//  Copyright (c) 2018 Cisco and/or its affiliates.
+//
+//  Licensed under the Apache License, Version 2.0 (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at:
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+//  Unless required by applicable law or agreed to in writing, software
+//  distributed under the License is distributed on an "AS IS" BASIS,
+//  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//  See the License for the specific language governing permissions and
+//  limitations under the License.
+
+package main
+
+import (
+	"log"
+	"time"
+
+	"github.com/ligato/cn-infra/agent"
+	"github.com/ligato/cn-infra/examples/new/greeter"
+	"github.com/ligato/cn-infra/logging/logrus"
+	"github.com/ligato/cn-infra/rpc/grpc"
+	"google.golang.org/grpc/examples/helloworld/helloworld"
+)
+
+func main() {
+	// Create agent
+	a := agent.NewAgent(
+		agent.Version("1.0.0"),
```

Comment:
This needs defaults badly.  I shouldn't have to specify version by default, or startup time by default... if those aren't present we should get default values for them.